Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -35,25 +48,35 @@ open class _QuotedChatMessageView<ExtraData: ExtraDataTypes>: _View, ThemeProvid
         }
     }
 
+    /// A Boolean value that checks if all attachments are empty.
+    open var isAttachmentsEmpty: Bool {
+        guard let content = self.content else { return true }
+        return content.message.fileAttachments.isEmpty
+            && content.message.imageAttachments.isEmpty
+            && content.message.linkAttachments.isEmpty
+            && content.message.giphyAttachments.isEmpty
+    }
+
     /// The container view that holds the `authorAvatarView` and the `contentContainerView`.
-    public private(set) lazy var containerView: ContainerStackView = ContainerStackView()
+    open private(set) lazy var containerView: ContainerStackView = ContainerStackView()
         .withoutAutoresizingMaskConstraints
 
     /// The avatar view of the author's quoted message.
-    public private(set) lazy var authorAvatarView: ChatAvatarView = components
+    open private(set) lazy var authorAvatarView: ChatAvatarView = components
```

Comment:
In `ChatMessageContentView` we expose the avatar size:
```swift
    open var messageAuthorAvatarSize: CGSize { .init(width: 32, height: 32) }
```

We might consider doing the same for:
```swift
override open func setUpLayout() {
        let authorAvatarSize = CGSize(width: 24, height: 24)
        let attachmentPreviewSize = CGSize(width: 34, height: 34)
        ...
}
```

Reply:
Done ✅